test/functions/getFirstExistingParentPath.spec.ts

```diff
@@ -1,24 +1,44 @@
+import { normalize } from 'node:path'
+
 import test from 'ava'
 import { PathLike } from 'fs'
 
 import getFirstExistingParentPath from '@/src/functions/getFirstExistingParentPath'
 import mockDependencies from '@/test/__helpers__/mockDependencies'
 
+const getDependencies = (parentPath: string) => mockDependencies({
```

Comment:
Maybe consider renaming that with the mocking notion in mind:

```suggestion
const getMockedDependencies = (parentPath: string) => mockDependencies({
```